src/lib/seam/devicedb/index.ts

```diff
@@ -1 +1,2 @@
-export const routes = {}
+export * from './public-models.js'
```

Comment:
Can we match the other entry point that collects schemas, e.g.?

```
import * as schemas from './stable/schemas.js'

export { schemas }
```

Reply:
@codetheweb

Reply:
sure, sorry I missed this